sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerIO.java

```diff
@@ -1594,6 +1595,8 @@ public abstract static class ReadChangeStream
     @Deprecated
     abstract @Nullable Double getTraceSampleProbability();
 
+    abstract @Nullable Duration getWatermarkRefreshRate();
```

Comment:
Do we need to set a default value for this when returnign the builder (see where rpcpriority default is set). I thought autobuilder complained about unset fields but perhaps nullable ones default to null?  In either case it might be better to handle the defaults consistently for the different fields.

Reply:
Added default value, where customers could set `Duration.ZERO` to disable the feature and get the old behavior.